### ISSUE TITLE
Unable to reference NetStandard project to Xamarin.Android and Xamarin.iOS projects

### ISSUE THREAD

**Issue by binoypatel** [dotnet/cli#2618]
First of, accept my apology if this is not a bug (using nuget I can reference the same project) or I am posting to wrong place, here is what is happening:

## Steps to reproduce
- Created a NetStandard project version 1.3, here is project.json:
{
  "supports": {},
  "dependencies": {
    "NETStandard.Library": "1.0.0-rc2-23910"
  },
  "frameworks": {
    "netstandard1.3": {}
  }
}
- Trying to reference above project in Xamarin.Android and/or Xamarin.iOS, using "Add reference->Solutions" and selecting above project from the list
- Here is what I am getting:
![image](https://cloud.githubusercontent.com/assets/4742252/14696143/9c4a5192-0746-11e6-8666-0be1e2154312.png)

## Expected  behavior
- Should be able to reference NetStandard 1.3 library to Xamarin.Android and Xamarin.iOS projects

## Actual behavior
- Unable to referenced NetStandard 1.3 library project to Xamarin.Android and Xamarin.iOS projects

## Environment data
.NET Command Line Tools (1.0.0-beta-001598)

Product Information:
 Version:     1.0.0-beta-001598
 Commit Sha:  7582649f88

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 Runtime Id:  win10-x64

**Comment by BeelzeBu** [comment]
Xamarin does not work with dotnet. There is a project to work with dnx and dnxcore5.0 rc1.

**Comment by piotrpMSFT** [comment]
@binoypatel if @BeelzeBu's response doesn't cover the question you're asking please reopen the issue. The fix here would not be coming from CLI, however. Xamarin itself would need to be updated to take advantage of netstandard.

/cc @richlander

**Comment by binoypatel** [comment]
@piotrpMSFT , I think I am not clear enough to report the issue. Ok let me rephrase it, I am using Visual studio 2015 update 2 with latest dotnet/cli installed. In the solution I have 3 projects:
ProjectA: this is netstandard project version 1.3
ProjectB: this is Xamarin.Android project
ProjectC: this is UWP project

I can reference ProjectA using Add Reference to ProjectC. If I do the same for ProjectB, it shows the error I attached in my first post (Screenshot from visual studio). Now, If I create Nuget package for ProjectA then I can reference it to ProjectB. This is telling me that If I want to reference netstandard library then I have currently no way but create a nuget package and reference in Xamarin.Android and Xamarin.iOS projects.

Let me know if is still not clear.

Thanks,
Binoy

**Comment by fubar-coder** [comment]
@binoypatel This issue is still closed, you may want to re-open it, when it's not resolved.

**Comment by binoypatel** [comment]
@fubar-coder  Thanks for heads-up but couldn't see an option to re-open this issue as I didn't close it, this issue is not resolved, please check my previous comment to repro, thanks

**Comment by fubar-coder** [comment]
I'm neither from Microsoft, nor an admin for this github repository :wink: . I suggest, that you create a new issue, explains that you were unable to open a new issue and link to the old issue.
